src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -674,6 +825,12 @@ export default {
 				this.speakingInterval = null
 			}
 		},
+
+		phoneCallStatus(value) {
+			if (value !== 'ringing' && value !== 'accepted') {
```

Comment:
```suggestion
			if (!value || (value !== 'ringing' && value !== 'accepted')) {
```

I'd check if that helps when we receive status `cleared`
Maybe also apply disabled state on DialpadPanel here?